### PR TITLE
[RFC][UN-6614] Improvements to uniq

### PR DESCRIPTION
**Pull request by mmelo**
Refers to: https://uniplaces.atlassian.net/browse/UN-6614

Several fixes to match UniQ's ember addon styles

### CODE REVIEW

File: src/styles/_components/_inputs.scss [hunk 1]
```diff
@@ -332,3 +332,54 @@ $checkbox-size: 24px;
...
+    @include justify-content-center();
+    @include align-items-center();
+    border: 2px solid color('input', 'border');
+    border-radius: 4px;
+
+    input,
+    &__value {
+        @include input();
+        font-size: em(20px);
+         margin: 0;
```
Comment (AndreJoaquim) [inline src/styles/_components/_inputs.scss]: remove extra space

File: src/styles/_components/_inputs.scss [hunk 2]
```diff
@@ -332,3 +332,54 @@ $checkbox-size: 24px;
...
+        max-width: 70%;
+        padding: 0;
+        padding-right: 36px;
+        border-left: 0;
+        line-height: $input-height;
+    }
+
+    &__currency {
+        color: color('text');
+    }
```
Comment (AndreJoaquim) [inline src/styles/_components/_inputs.scss]: new line

**Comment by AndreJoaquim** [comment]
Reviewd

File: src/styles/_components/_inputs.scss (hunk 2 above)
Comment (mmelo) [inline src/styles/_components/_inputs.scss]: done

**Comment by mmelo** [comment]
@AndreJoaquim @raffo0707 fixed

**Pull request merged**